generation/WinSDK/RecompiledIdlHeaders/um/winternl.h

```diff
@@ -485,6 +482,29 @@ NtOpenFile (
     IN ULONG OpenOptions
     );
 
+//
+// use the Win32 API instead
+//     CreateNamedPipeA
+//
+__kernel_entry NTSTATUS
+NTAPI
+NtCreateNamedPipeFile(
```

Comment:
Won't this file get clobbered the next time we run UpdateSDK with whatever is really in the SDK?

is this function definition available in a Windows SDK header that we could use / include instead?

Reply:
Docs say it lives in `ntioapi.h` which isn't public. Agree, we should minimally move it to a new header (e.g., `\generation\WinSDK\AdditionalHeaders\ntioapi.h`)